nectar/downloaders/threaded.py

```diff
@@ -293,9 +293,7 @@ def _fetch(self, request):
             report.download_skipped()
 
         except requests.ConnectionError as e:
-            _logger.warning("Connection Error - {url} could not be reached.".format(
-                url=request.url)
-            )
+            _logger.warning(str(e))
             self.failed_netlocs.add(netloc)
```

Comment:
WELL IM A PLUS ONE BUT MOSTLY BECAUSE YOU MADE A FUNCTION THAT WAS LIKE A MILLION LINES LONG BE NINE HUNDRED AND NINETY NINE THOUSANE NINE HUNDRED AND NINETY EIGHT LINES LONG AND I THINK THAT IS AN IMPROVEMENT NICE WORK BRO.